docs/app/pages/components/components-sections/notifications/badge/badge.component.html

```diff
@@ -0,0 +1,158 @@
+<div class="row badge-row">
+    <div class="col-sm-3 col-xs-6">
```

Comment:
I do think it would be better if these occupied the full width (`col-md-6`) and controls centered within the column.

Reply:
That was at the request of Roland. He mentioned it should be consistent with the other components.

Reply:
Did he give any example of what this style is consistent with? Looking through the widgets category we have examples that are [centered](https://pages.github.houston.softwaregrp.net/caf/ux-aspects-micro-focus/#/ui-components/widgets#circular-progress) and examples that are [left-aligned](https://pages.github.houston.softwaregrp.net/caf/ux-aspects-micro-focus/#/ui-components/widgets#spinner) so I don't really think we have anything to be consistent with.

Reply:
I see what you mean. The majority of examples in widgets -> Microfocus and all in the notifications -> OpenSource are left aligned . Does this mean that the others should be updated for consistency sake, so all are left aligned? Or I can change this example back.

Reply:
No, I think it should be two columns, center aligned within each.

Reply:
Let's think of it this way, if you left-align then you would have to add a significant left margin to allow testing of the "before" alignments with potentially large strings, so it's not actually going to be aligned with anything. (For that reason you should probably add `col-xs-12` in there as well, just in case.)